authentication/managing_cloud_provider_credentials/about-cloud-credential-operator.adoc

```diff
@@ -0,0 +1,90 @@
+[id="about-cloud-credential-operator"]
+= About the Cloud Credential Operator
+include::modules/common-attributes.adoc[]
+:context: about-cloud-credential-operator
+:FeatureName: Support for Amazon Web Services Secure Token Service (AWS STS)
+toc::[]
+
+//Links that point to the entry in the RH Operators ref should now point here
+
+The Cloud Credential Operator (CCO) manages cloud provider credentials as Kubernetes custom resource definitions (CRDs). The CCO syncs on `CredentialsRequest` custom resources (CRs) to allow {product-title} components to request cloud provider credentials with the specific permissions that are required for the cluster to run.
```

Comment:
s/Kubernetes custom resource definitions/custom resource definitions ??

I don't see any other references to _Kubernetes custom resource definitions_ using an Atom search